launch/launch/actions/execute_local.py

```diff
@@ -464,9 +480,35 @@ def printer(context, msg, timeout_substitutions):
             ],
             cancel_on_shutdown=False,
         )
+        def kill_subprocesses(
+            context,
+            timeout_substitutions,
+            children=psutil.Process(
+                self._subprocess_transport.get_pid()).children(recursive=True)
```

Comment:
Is it possible for `self._subprocess_transport.get_pid()` to return None or something like that? Maybe this should be broken into steps with proper error handling in place?

Reply:
Any comment on this one?

Reply:
sorry, I didn't see that before.
I can check if it can return `None` or something else if the process died.

Reply:
See code after https://github.com/ros2/launch/pull/632/commits/85ea3a90682751c3c817c197afb0d45d1173e807